### ISSUE TITLE
Southpole V2 build (Windows only)

### ISSUE THREAD

**Issue by fractalgee** [gbrandt1/southpole-vcvrack#55]
@SteveRussel33's v1 port branch also compiles fine for v2 (Only change needed is to change version number to 2.0.0 in plugin.json, then build)

Here is the build I did and tested on my build of v2, nothing amiss so far but not excessively tested so let us hear what issues others may find....

Remove .zip from filename, then put into v2 plugin folder and off you go

[southpole-2.0.0-win.vcvplugin,zip](https://github.com/gbrandt1/southpole-vcvrack/files/7243192/southpole-2.0.0-win.vcvplugin.zip)

Some routine for the parasites version:

[Southpole-parasites-2.0.0-win.vcvplugin.zip](https://github.com/gbrandt1/southpole-vcvrack/files/7244298/Southpole-parasites-2.0.0-win.vcvplugin.zip)

**Comment by fractalgee** [comment]
Careful: Will only work on local build of Rack v2 as of today or newer......  Will not load on posted beta build until that gets updated......

**Comment by flyingLowSounds** [comment]
Here is a mac build for VCV Rack Community Edition 2.git.b04e4117. It seems fine but not extensively tested.

[southpole-2.0.0-mac.vcvplugin](https://github.com/flyingLowSounds/VCV-Rack-v2-Testing-Plugins/blob/main/southpole-2.0.0-mac.vcvplugin)

**Comment by flyingLowSounds** [comment]
Parasites build for VCV Rack Community Edition 2.git.b04e4117...

[Southpole-parasites-2.0.0-mac.vcvplugin](https://github.com/flyingLowSounds/VCV-Rack-v2-Testing-Plugins/blob/main/Southpole-parasites-2.0.0-mac.vcvplugin)

**Comment by nay-seven** [comment]
Thanks to both of you for these updates and Mac Builds ! no issues currently on Mac/Mojave OS

**Comment by Sanveannils** [comment]
Hello, i have remove the .zip, place them in the /document / rack2 / plugins, restart but didnt work, on pc,
maybe i have miss something? :)

**Comment by SteveRussell33** [comment]
> @SteveRussel33's v1 port branch also compiles fine for v2 (Only change needed is to change version number to 2.0.0 in plugin.json, then build)
> 
> Here is the build I did and tested on my build of v2, nothing amiss so far but not excessively tested so let us hear what issues others may find....
> 

My fork is not for building - it’s for my v1 PR on the original 0.6 repo. Building from that will not include all the PRs.

I only made my windows v1 build available as binary, not the repo. https://github.com/gbrandt1/southpole-vcvrack/issues/36#issuecomment-544230931

Therefore it’s not a complete v1 fork and certainly NOT for building against v2! Forks are not always for complete ports, a lot are used to host PR commits as in this case.

THE BINARIES POSTED HERE ARE INCOMPLETE.

This issue should either be closed or the binaries removed.

**Comment by fractalgee** [comment]
> > @SteveRussel33's v1 port branch also compiles fine for v2 (Only change needed is to change version number to 2.0.0 in plugin.json, then build)
> > Here is the build I did and tested on my build of v2, nothing amiss so far but not excessively tested so let us hear what issues others may find....
> 
> My fork is not for building - it’s for my v1 PR on the original 0.6 repo. Building from that will not include all the PRs.
> 
> I only made my windows v1 build available as binary, not the repo. [#36 (comment)](https://github.com/gbrandt1/southpole-vcvrack/issues/36#issuecomment-544230931)
> 
> Therefore it’s not a complete v1 fork and certainly NOT for building against v2! Forks are not always for complete ports, a lot are used to host PR commits as in this case.
> 
> THE BINARIES POSTED HERE ARE INCOMPLETE.
> 
> This issue should either be closed or the binaries removed.

hmmm, That was the one I built first, However I just rebuilt it from @flyingLowSounds repo, that one I am also not sure if feature complete, but does fully work in v2/ So, I will close this one, and we shall wait until someone hosts a feature complete repro (if his isn't) as well as the parasites branch from netboy3 so we can have both at release. Parasites Clouds port is still the only one that can do reverse granular.....

**Comment by SteveRussell33** [comment]
Unless you are including ALL the PRs https://github.com/gbrandt1/southpole-vcvrack/pulls ANY clone build from v1 branches will be incomplete and not have the entire list of fixes.
It may compile and "work" in v2 but it still is incomplete in terms of fixes and API additions.

Southpole Parasites' Smoke is not the only one that does reverse granular. The other is Arable Instruments' "Neil" module which had a v1 port done 2 years ago. I'll be working on that once my Southpole work is done - again this will be a separate v2 repo.

**Comment by fractalgee** [comment]
never used the Neil, so ok, I stand corrected. I still like the formfactor
of the SP better, Neil is quite gigantic compared to that but alas,
whatever works. Now let's stop the irritated tone please, I heard you loud
and clear and never intended to ruffle feathers

**Comment by flyingLowSounds** [comment]
> Now let's stop the irritated tone please, I heard you loud
> and clear and never intended to ruffle feathers

I agree

**Comment by SteveRussell33** [comment]
It's not being irritated or ruffled, it's being concerned that one of you is replicating work I've already done (and continue to work on) people need to be aware it's for testing only i.e. patches shouldn't be saved with that version and changes in the near future may break them.

The other is promoting on the community forum to build from a repo that is not for building. The fact that both of you didn't pick this up is the concern. Now people have binaries that aren't complete or inherently ready for v2. That's why I had to put out my message more than once so that people will pick it up and not think to use them in a production setting.

They will be ready for v2 release soon.

**Comment by veryfungi** [comment]
> Some routine for the parasites version:
> 
> [Southpole-parasites-2.0.0-win.vcvplugin.zip](https://github.com/gbrandt1/southpole-vcvrack/files/7244298/Southpole-parasites-2.0.0-win.vcvplugin.zip)

Hi Fractalgee, I got my southpole modules from my friend Dan awhile ago, but I was hoping to get the parasites version of smoke. I tried your link, but the modules do not show up. I removed the zip from the filename and put it into the plugins folder. It is now listed in the folders, but the modules do not show up in VCV. Is there anything I can do?

**Comment by obo-works** [comment]
> Here is a mac build for VCV Rack Community Edition 2.git.b04e4117. It seems fine but not extensively tested.
> 
> [southpole-2.0.0-mac.vcvplugin](https://github.com/flyingLowSounds/VCV-Rack-v2-Testing-Plugins/blob/main/southpole-2.0.0-mac.vcvplugin)

Are the Mac files no longer available?  Definitely curious to check out these modules (along with Parasites)
